posthog/models/cohort.py

```diff
@@ -146,6 +147,13 @@ def properties(self):
 
         return PropertyGroup(PropertyOperatorType.AND, cast(List[Property], []))
 
+    @cached_property
```

Comment:
using `cached_property` here seems a bit dicey, since it doesn't seem like this will update when a cohort updates to not have `behavioral` filters.